src/frontend/context/SettingsContext.tsx

```diff
@@ -1,30 +1,31 @@
-// @flow
 import * as React from "react";
 import createPersistedState from "../hooks/usePersistedState";
 
 // Increment if the shape of settings changes, but try to avoid doing this
 // because it will reset everybody's settings back to the defaults = bad :(
-const STORE_KEY = "@MapeoSettings@2";
+const STORE_KEY = "@MapeoSettings@3";
```

Comment:
just pointing this out since I'm sure we'd want to consider this heavily before merging